src/data/finset/lattice.lean

```diff
@@ -411,15 +409,18 @@ lemma sup_inf_distrib_right (s : finset ι) (f : ι → α) (a : α) :
 by { rw [_root_.inf_comm, s.sup_inf_distrib_left], simp_rw _root_.inf_comm }
 
 protected lemma disjoint_sup_right : disjoint a (s.sup f) ↔ ∀ i ∈ s, disjoint a (f i) :=
-by simp only [disjoint_iff, sup_inf_distrib_left, sup_eq_bot_iff]
+by simp only [disjoint_iff, sup_inf_distrib_left, finset.sup_eq_bot_iff]
 
 protected lemma disjoint_sup_left : disjoint (s.sup f) a ↔ ∀ i ∈ s, disjoint (f i) a :=
-by simp only [disjoint_iff, sup_inf_distrib_right, sup_eq_bot_iff]
+by simp only [disjoint_iff, sup_inf_distrib_right, finset.sup_eq_bot_iff]
+
+lemma sup_inf_sup : s.sup f ⊓ t.sup g = (s ×ˢ t).sup (λ i, f i.1 ⊓ g i.2) :=
```

Comment:
I assume the n-ary version is not worth the trouble.

Reply:
Done anyway :^)